### ISSUE TITLE
Question about baseline on ImageNet

### ISSUE THREAD

**Issue by EdwardLiuYu** [he-y/filter-pruning-geometric-median#60]
Hi, I find that in your experiment the baseline on ImageNet is different from the result reported in torch.utils.model_zoo. For example, the top1 of resnet18 in model_zoo is 69.76 but in your paper it is 70.28. Could you explain the reason?

**Comment by he-y** [comment]
We use 100 epochs for training, see [the link](https://github.com/he-y/filter-pruning-geometric-median#why-use-100-epochs-for-training).

**Comment by EdwardLiuYu** [comment]
So, the pre-trained model for pruning is based on official baseline of pytorch or your own baseline?

**Comment by he-y** [comment]
The pre-trained model for pruning is based on 90-epoch official baseline of pytorch.
The retraining epoch is 100.
so we report the 100-epoch accuracy as the baseline.

**Comment by EdwardLiuYu** [comment]
Ok, I get it. Thanks for the answers.

**Comment by he-y** [comment]
Thanks for your interest.
